### PR TITLE
Add initial transcription page layout

### PR DESCRIPTION
**Pull request by ghost**
The initial layout for the transcription / project page for the app.

Only basic and very open to change. Layout currently supports resizing of the video component.

### CODE REVIEW

File: src/renderer/components/StandardButton.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Button } from '@mui/material';
+
+const StandardButton = Button;
+
```
Comment (chloebrett) [inline src/renderer/components/StandardButton.tsx:4]: Assume this is a place holder to add styling later?

**Review by chloebrett — COMMENTED** [review]
Looks snazzy

File: src/renderer/pages/Project.tsx [hunk 2]
```diff
@@ -24,19 +26,54 @@ const ProjectPage = () => {
...
+          </Box>
+        </Stack>
+        <Box sx={{ width: '2px', backgroundColor: 'gray' }} />
+        <Stack justifyContent="center" sx={{ width: 'fit-content' }}>
+          <Box sx={{ width: '400px', height: '280px', backgroundColor: 'red' }}>
+            video
+          </Box>
+          <div>
+            <div>{saveButton}</div>
+            <div>{openButton}</div>
```
Comment (fwang1703) [inline src/renderer/pages/Project.tsx:68]: What's the correlation of these buttons to the hi-fi designs? Thought we were having save and open in the OS menu bar?

File: src/renderer/components/StandardButton.tsx (hunk 1 above)
Comment (ghost) [reply src/renderer/components/StandardButton.tsx:4]: Yes. Definitely. Just trying to get everyone using a standardised button and someone can come in and do the props / forwarding / styling, etc.

**Review by fwang1703 — APPROVED** [review]

File: src/renderer/pages/Project.tsx (hunk 2 above)
Comment (ghost) [reply src/renderer/pages/Project.tsx:68]: I believe these buttons are just placeholders for team Donkey. Included for now but will likely be removed when functionality moved into the file menu.

**Pull request merged**